### ISSUE TITLE
Hover states for Close & Settings icons in modals are not centered

### ISSUE THREAD

**Issue by cdonovan** [home-assistant/frontend#6446]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

As of the 0.113 release, the hover background circles are not initially centered on the icons. It does seem that after an interaction with the icon, however, the hover position remains corrected (until the modal gets re-rendered).

Attached is a screenshot of the issue.  It can be seen for any of the icons in the modal header.
![hover-circle-position](https://user-images.githubusercontent.com/1992730/88233412-7f9f6000-cc2c-11ea-83ec-c08a3af81ed0.png)

Here is a video of it happening for a bit more info - https://streamable.com/hljrwf

## Expected behavior

The hover state background circles should be centered on the icons.

## Steps to reproduce

1. Open an entity modal by clicking on an entity
2. Hover your cursor over an icon, such as Close (X) or Settings (gear or slider icons) and observe that the hover circle is not centered.

## Environment

- Home Assistant release with the issue: 0.113.0
- Last working Home Assistant release (if known): Unknown
- Browser and browser version: Chrome 84.0.4147.89, Firefox 78.0.2 and Safari 13.1.2
- Operating system:Mac 10.14.6

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

**Comment by SeanPM5** [comment]
Duplicate of https://github.com/home-assistant/frontend/issues/6285

Bram's fix was merged a few hours ago, but it's waiting on a release (on Google's side), not sure when that will be.

**Comment by cdonovan** [comment]
Thanks @SeanPM5, I didn't notice this one, I guess my searching for "modal icon" was a bit off, since these are called dialogs in mwc.  Appreciate the quick response!.
